front/src/common/api/osrdEditoastApi.ts

```diff
@@ -42,24 +42,115 @@ const osrdEditoastApi = generatedEditoastApi.enhanceEndpoints({
       // we don't want to invalidate the trainschedule tag here to prevent multiple calls
       invalidatesTags: ['timetable', 'scenarios'],
     },
-    // Invalidate the children count and last update timestamp
+
+    // Project handling
+    getProjects: {
+      providesTags: (result) => [
+        'projects',
+        { type: 'projects', id: 'LIST' },
+        ...(result?.results || []).map((project) => ({
+          type: 'projects' as const,
```

Comment:
Sometimes we're using `as const`, sometimes not - do we need it?

Reply:
As seen this morning, ts requires the `as const` in the maps

Reply:
Yeah, plus this is what's used in the official docs!